ES/ESmy018.xml

```diff
@@ -32,86 +32,185 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-018</idno>
                </msIdentifier>
                <msContents>
-                  <summary>I) Taʾammǝra Māryām (fols. 3ra-43vb)&#xD; I-1) Introductory exhortation
+                  <summary>
+                    I) Taʾammǝra Māryām (fols. 3ra-43vb); I-1) Introductory exhortation
                      (fols. 3ra-4va)&#xD; I-2) Taʾammǝra Māryām: 30 miracles (fols. 4va-43vb)&#xD; II)
                      Discourse by John Chrysostom on the life and decollation of John the Baptist
-                     (fol. 44ra-60va) &#xD; III) Homily on the finding of the head of John the
-                     Baptist (fols. 60vb-65ra)&#xD; IV) Homily by John Chrisostom on the glory of
-                     John the Baptist (65ra-74ra)&#xD; &#xD; Additiones 1-2; colophons
-                     1-2.</summary>
-                  <msItem xml:id="ms_i1">
```

Comment:
here part of summary seems to have stayed and part disappeared?

Reply:
Removed